spec/queries/get_recruited_application_choices_spec.rb

```diff
@@ -72,8 +72,8 @@
         :application_choice,
         :with_deferred_offer_previously_recruited,
         application_form: build(:application_form, recruitment_cycle_year: '2021'),
-        updated_at: Time.zone.now + 1.day,
       )
+      deferred_application.update(updated_at: Time.zone.now + 1.day)
```

Comment:
Needs to be changed as the test setup (adding an offer) was overwriting the updated_at feild.